src/lib/ngsi/ContextRegistrationResponse.h

```diff
@@ -39,8 +39,11 @@
 */
 typedef struct ContextRegistrationResponse
 {
-  ContextRegistration   contextRegistration;    // Mandatory
-  StatusCode            errorCode;              // Optional
+  ContextRegistration   contextRegistration;        // Mandatory
+  StatusCode            errorCode;                  // Optional
+
+  std::string           forwardingMode;             // During forwarding, this field is used to record the 'format' from the registration that provoked
```

Comment:
Better to use "format" or "providerFormat".

Reply:
Not sure where I got that name from ...
So, "providerFormat"

Reply:
Fixed in c522e4e